components/moonstone/authentication/Text/index.jsx

```diff
@@ -0,0 +1,12 @@
+export default function Text (props){
```

Comment:
```suggestion
export default function Text ( {padding, href }){
```
It is more clean to destructure the props in the function signature.